externs/shaka/player.js

```diff
@@ -813,7 +814,12 @@ shaka.extern.PersistentSessionMetadata;
  *   <i>By default (''), do not check the HDCP version.</i><br>
  *   Indicates the minimum version of HDCP to start the playback of encrypted
  *   streams. <b>May be ignored if not supported by the device.</b>
- *
+ * @property {boolean} ignoreDuplicateInitData
+ *   <i>Defaults to false on Tizen 2, and true for all other browsers.</i><br>
+ *   When true indicate that the player doesn't ignore duplicate init data.
```

Comment:
I think it should be "When true indicates that the player does ignore duplicate init data". (Without negation)